New fix

ayan-biswas opened the pull request.

edwardcwang commented on bag/math/interpolate.py:
```diff
@@ -169,45 +170,51 @@ def integrate(self, xstart, xstop, axis=-1, logx=False, logy=False, raw=False):
         # get all input sample points we need to integrate.
         plist = []
-        integ_x = None
+        integ_x = calculate_integ_x()  # type: np.ndarray
```
Thanks for the catch - fixed.

pvnbhargava commented on bag/layout/digital.py:
```diff
@@ -286,21 +292,24 @@ def add_std_instance(self, master, inst_name=None, loc=(0, 0), nx=1, ny=1,
 
-        spx *= col_pitch
-        spy *= row_pitch
+        spx_new = spx * col_pitch
```
Just curious, is there a specific reason to store `spx * col_pitch` into a new variable? It seems like the existing code functions fine as is...

edwardcwang replied:
`spx` is declared as an `int but the `*=` operator (called with a float) tries to change it to a `float` (col_pitch is a float) which is inconsistent with `spx`'s declared type of `int`.

pvnbhargava commented on bag/layout/util.py:
```diff
@@ -567,7 +567,7 @@ def __init__(self, bbox, nx=1, ny=1, spx=0, spy=0, unit_mode=False):
 
     def __iter__(self):
-        # type: () -> Generator[BBox]
+        # type: () -> Iterator[BBox]
```
Shouldn't this stay as a Generator type? We are yielding the `bbox` instead of explicitly writing a `__next__()` method...

edwardcwang replied:
The Generator type is much more general ( `Generator[YieldType, SendType, ReturnType]`). Since this function is returned *by* an `__iter__` I think the type makes more sense as an `Iterator`. Moreover it doesn't send or return any types...

Commit:
1. Connect by name in schematic netlist; 2. Fix gds passing in run_rcx

Commit:
fixing bug with Calibre PEX extraction checking

Commit:
resolving problem in calibre view creation

Commit:
adding optional rcx_params capability to pex extraction runset in calibre.py

Commit:
changing error message for PEX to support old versions of Calibre

boblinchuan committed:
Copying over changes, including power fill and poly path

boblinchuan committed:
Removing int casting

ayan-biswas committed:
Merge pull request #9 from boblinchuan/bob

Copying over changes, including power fill and poly path

Commit:
update calibre view creation API to support old versions of Calibre

Commit:
minor modification in calibre view creation to avoid errors

Commit:
supporting different extraction types in QRC

Commit:
Merge branch 'new_fix' of github.com:ucb-art/BAG_framework into new_fix

Commit:
explicitly remove old files during Calibre-PEX extraction flow

Commit:
Merge branch 'new_fix' of github.com:ucb-art/BAG_framework into new_fix

mkosunen committed:
Merge pull request #1 from ucb-art/new_fix

New fix

mkosunen committed:
Remove master_key also when parameters given

In class SkillInterface(DbAccess)
If you provide parameters for primitive instance, the master:key was not
removed, which caused skill interface to fail. This fixes it.

ayan-biswas committed:
Merge pull request #10 from aalto-ele/fix_skill_interface

Fix skill interface for master:key not existing
